content/blog/introducing-python-sdk-for-dscc.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Introducing Python SDK for DSCC
+date: 2023-03-29T13:45:50.979Z
+priority: 0
+author: Anusha Y and Sijeesh Kattumunda
+authorimage: /img/Avatar1.svg
+disable: false
+---
+In my previous blog, I discussed how to generate a Software Development Kit (SDK) from the Data services cloud console (DSCC) Open API spec using a third-party tool called OpenAPI generator. Today, I am going to talk more in detail on the Python SDK generated using the tool. As we progress, we will discuss the implementation aspects, and then take up a use-case to better understand how to use the SDK. 
```

Comment:
In my previous blog...
You may want to hotlink "In my previous blog" to reference the blog post https://developer.hpe.com/blog/get-started-building-dscc-api-client-libraries-for-python-using-openapi-generator/

To make a reference of the previous post, please refer to tip #4 from the guide here: https://github.com/hpe-dev-incubator/hpe-dev-portal/blob/master/docs/ContributorGuide-v2.md#tips-and-trick-using-the-cms-editor

[In my previous blog](https://developer.hpe.com/blog/get-started-building-dscc-api-client-libraries-for-python-using-openapi-generator/), I discussed how...